assets/src/modules/config/Options.js

```diff
@@ -43,25 +44,26 @@ export class OptionsConfig  extends BaseObjectConfig {
-     * @param {boolean}  [cfg.hideProject]              - is the project hidden in user interface ? Only services are available.
-     * @param {number}   [cfg.wmsMaxHeight]             - the image max height for WMS GetMap request
-     * @param {number}   [cfg.wmsMaxWidth]              - the image max width for WMS GetMap request
-     * @param {boolean}  [cfg.fixed_scale_overview_map] - does the Overview map have fixed scale ?
-     * @param {boolean}  [cfg.use_native_zoom_levels]   - does the map use native zoom levels ?
-     * @param {boolean}  [cfg.hide_numeric_scale_value] - does the scale line hide numeric scale value ?
-     * @param {boolean}  [cfg.hideGroupCheckbox]        - are groups checkbox hidden ?
+     * @param {object}   cfg                                      - the lizmap config object for options
+     * @param {number[]} cfg.bbox                                 - the project and web services max extent
+     * @param {number[]} cfg.initialExtent                        - the map extent at the loading page
+     * @param {number[]} cfg.mapScales                            - the map scales
+     * @param {number}   cfg.minScale                             - the map's min scale
+     * @param {number}   cfg.maxScale                             - the map's max scale
+     * @param {object}   cfg.projection                           - the web map projection
+     * @param {number}   cfg.pointTolerance                       - the point tolerance for QGIS Server WMS GetFeatureInfo request
+     * @param {number}   cfg.lineTolerance                        - the line tolerance for QGIS Server WMS GetFeatureInfo request
+     * @param {number}   cfg.polygonTolerance                     - the polygon tolerance for QGIS Server WMS GetFeatureInfo request
+     * @param {string}   cfg.popupLocation                        - the popup location in the User interface: dock, bottom-dock, right-dock, mini-dock, map
+     * @param {string}   cfg.datavizLocation                      - the popup location in the User interface: dock, bottom-dock, right-dock
+     * @param {boolean}  [cfg.hideProject]                        - is the project hidden in user interface ? Only services are available.
+     * @param {number}   [cfg.wmsMaxHeight]                       - the image max height for WMS GetMap request
+     * @param {number}   [cfg.wmsMaxWidth]                        - the image max width for WMS GetMap request
+     * @param {boolean}  [cfg.fixed_scale_overview_map]           - does the Overview map have fixed scale ?
+     * @param {boolean}  [cfg.use_native_zoom_levels]             - does the map use native zoom levels ?
+     * @param {boolean}  [cfg.hide_numeric_scale_value]           - does the scale line hide numeric scale value ?
+     * @param {boolean}  [cfg.hideGroupCheckbox]                  - are groups checkbox hidden ?
+     * @param {boolean}   [cfg.wms_single_request_for_all_layers] - are layers loaded as single WMS image ?  
```

Comment:
```suggestion
     * @param {boolean}  [cfg.wms_single_request_for_all_layers]  - are layers loaded as single WMS image ?  
```
Just a bit of formatting.